extensions/vscode/src/views/homeView.ts

```diff
@@ -953,6 +963,9 @@ export class HomeViewProvider implements WebviewViewProvider, Disposable {
         if (value.length === 0) {
           return "Secret names cannot be empty.";
         }
+        if (existingKeys.has(value)) {
+          return "There is already an environment variable with this name. Secrets and environment variable cannot share a name.";
```

Comment:
```suggestion
          return "There is already an environment variable with this name. Secrets and environment variable names must be unique.";
```

Slightly more direct?

Reply:
Adjusted. Great suggestion 👍